chainio/types.go

```diff
@@ -0,0 +1,37 @@
+package chainio
+
+import (
+	"encoding/hex"
+	"errors"
+	"strings"
+)
+
+const (
+	// We use uint8 to count the number of quorums, so we can have at most 255 quorums,
+	// which means the max ID can not be larger than 254 (from 0 to 254, there are 255
+	// different IDs).
+	MaxQuorumID = 254
+)
+
+type QuorumID = uint8
+
+type OperatorID = [32]byte
+
+func GetOperatorHex(id OperatorID) string {
```

Comment:
Right now the `OperatorID` type is just an alias instead of a true type (`type OperatorID = [32]byte` vs `type OperatorID [32]byte`). This seemed simpler for portability as this type is used in many packages. It's inconventient to have to explicitly convert each operator id into `chainio.OperatorID`. 

The downside is that you can't define new methods on aliased types.